workflow/github.md

```diff
@@ -1,2 +1,138 @@
 # Working on GitHub
 
+This document outlines the rules to contribute and to create issues to a project on the organisation [Canonical-websites](https://github.com/canonical-websites/)
+
+## Table of content
+
+* [How can I contribute?](#how-can-i-contribute-)
+  + [Report Bugs](#report-bugs)
+  + [Suggest Enhancements](#suggest-enhancements)
+  + [Work on an issue](#work-on-an-issue)
+* [Pull Requests](#pull-requests)
+  + [First contribution](#first-contribution)
+  + [Styleguides](#styleguides)
+    - [Commits messages](#commits-messages)
+    - [HTML](#html)
+    - [JS](#js)
+    - [Stylesheets](#stylesheets)
+    - [Python](#python)
+    - [Documentation](#documentation)
+  + [Merging process](#merging-process)
+* [Templates](#templates)
+  + [Issues template](#issues-template)
+  + [Pull-request template](#pull-request-template)
+
+## How can I contribute?
+
+In the Canonical web-sites organisation we work with Github. All types of work are tracked as GitHub issues. You will find in this section how you can contribute to this organisation.
+
+### Report Bugs
+
+Well done you have found a bug! Go to the corresponding project, it should be in the [Canonical websites organisation](https://github.com/canonical-websites/) and open an issue. These are the steps you need follow to add an issue:
+* **Title:** should describe your feature/bug
```

Comment:
This feels like repetition - couldn't we just link them down to the issue template?

Reply:
I agree, this will go out of date of we update the issue template. Let's just link to it.

Reply:
the template is only here for the description. Do we have a Title template? Do we need one? I would redirect directly to the template section if we want to set one.